docs/Changelog.md

```diff
@@ -9531,6 +9531,49 @@ This version of the operator has been available since version 10 of the default
 <dd>Constrain output mask types to boolean tensors.</dd>
 </dl>
 
+### <a name="IsNaN-10"></a>**IsNaN-10**</a>
+
+  Map elements in float tensor to either true or false based on its
+      attributes.
+
+#### Version
+
+This version of the operator has been available since version 10 of the default ONNX operator set.
+
+#### Attributes
+
+<dl>
+<dt><tt>detectnegativeinfinity</tt> : int (default is 0)</dt>
```

Comment:
Use the same style for attribute naming as the rest of ONNX. The above should be 'detect_negative_infinity'.

Reply:
Test files are added and now the strategy becomes
- Not to touch IsNaN
- Add IsInf instead.